addons/mail/static/src/components/messaging_menu/messaging_menu.xml

```diff
@@ -54,11 +54,10 @@
                             classNameObj="{ 'o-mobile': messaging.device.isMobile }"
                             filter="messagingMenu.activeTabId"
                         />
-                        <t t-if="messaging.device.isMobile">
+                        <t t-if="messaging.device.isMobile and messagingMenu.mobileMessagingNavbar">
                             <MobileMessagingNavbar
                                 className="'o_MessagingMenu_mobileNavbar'"
```

Comment:
ideally move classname to new model

Reply:
I don't get what you mean by "move classname to new model" in this context?

Reply:
I prefer to keep structural classnames in the template for styling purposes, or it's gonna be very hard to maintain with current code.

Note that the end goal is to remove the structural classnames.